pkgs/sketch_pad/lib/editor/editor.dart

```diff
@@ -5,38 +5,42 @@
 // ignore_for_file: avoid_web_libraries_in_flutter
 
 import 'dart:async';
-import 'dart:html' as html;
+import 'dart:js_interop';
 import 'dart:math' as math;
 import 'dart:ui_web' as ui_web;
 
-import 'package:codemirror/codemirror.dart';
-import 'package:codemirror/hints.dart';
 import 'package:dartpad_shared/services.dart' as services;
 import 'package:flutter/material.dart';
+import 'package:web/helpers.dart' as web_helpers;
+import 'package:web/web.dart' as web;
 
 import '../model.dart';
+import 'codemirror.dart';
 
 final Key _elementViewKey = UniqueKey();
 
-html.Element _codeMirrorFactory(int viewId) {
-  final div = html.DivElement()
+web.Element _codeMirrorFactory(int viewId) {
+  final div = web_helpers.createElementTag('div') as web.HTMLDivElement
```

Comment:
This is fine, but just curious: Is there any benefit of this over `web.document.createElement('tag')`?

Reply:
Nope the helpers package is just a pass-through API for convenience.